common/src/test/java/io/confluent/developer/TestUtils.java

```diff
@@ -20,10 +20,16 @@ public static LocalDateTime yyyy_MM_dd(String dateString) {
     }
 
     public static List<Row> rowObjectsFromTableResult(TableResult tableResult) throws Exception {
+        return rowObjectsFromTableResult(tableResult, Integer.MAX_VALUE);
+    }
+
+    public static List<Row> rowObjectsFromTableResult(TableResult tableResult, int maxRows) throws Exception {
```

Comment:
added to work around `CloseableIterator.hasNext` hanging for the `MATCH_RECOGNIZE` query. i.e., after reading the two rows, the next call to `hasNext` [here](https://github.com/confluentinc/tutorials/blob/master/common/src/test/java/io/confluent/developer/TestUtils.java#L25) hangs. Tried adding `LIMIT 2` to query but it still hangs with that.